### ISSUE TITLE
exporter: unable to use IPLDResolver .get(path) for inexistent paths

### ISSUE THREAD

**Issue by pgte** [ipfs/js-ipfs-unixfs-engine#177]
While developing [the subtree support feature](https://github.com/ipfs/js-ipfs-unixfs-engine/pull/175), I tried using the IPLD resolver to solve paths inside CBOR / non-UnixFS nodes, and it hanged, never returning. [This line](https://github.com/ipfs/js-ipfs-unixfs-engine/blob/c060b53f1c0510179c5644a91ee37db8c087ca21/src/exporter/index.js#L50) should be something like:

```js
dag.get(cid, path.rest, (err, node) => {
   // now we should have node.remainderPath to resolve further...
```

**Comment by diasdavid** [comment]
ping @vmx

**Comment by vmx** [comment]
The code changed a lot since the bug was reported (I think it can be closed). This is a UnixFS issue, not an IPLD one, hence assigning to @achingbrain.

**Comment by achingbrain** [comment]
I don't think this is a problem any more - I've implemented it in ipfs/js-ipfs-unixfs-exporter#1 but it needs a [change](https://github.com/ipld/js-ipld/pull/181) made to `js-ipld` first.
